include/deal.II/grid/grid_generator.h

```diff
@@ -984,11 +984,11 @@ namespace GridGenerator
    * refined cells.
    *
    * @note The function copies the material ids of the cells of the two input
-   * triangulations into the output triangulation but it currently makes no
-   * attempt to do the same for boundary ids. In other words, if the two
-   * coarse meshes have anything but the default boundary indicators, then you
-   * will currently have to set boundary indicators again by hand in the
-   * output triangulation.
+   * triangulations into the output triangulation. If @p copy_manifold_ids is
+   * activated, manifold ids will be copied. Currently, boundary_ids are never
```

Comment:
activated -> set to @p true

Also, remove `currently` -- we should be documenting the current state of the library, not make reference to past or future behavior